SmRtAPI/SmRtAPI/V2/SmRtApi.cs

```diff
@@ -98,7 +137,7 @@ public void Run()
                         }, CancelToken, TaskCreationOptions.LongRunning, TaskScheduler.Default);
 
                         transcriptionComplete.WaitOne();
-                        Task.WaitAll(t1, t2);
+                        await Task.WhenAny(t1, t2);
```

Comment:
Maybe because the writing loop will probably finish first and it might finish before the reading loop has got everything and executed all the callbacks?

Reply:
I think this went in by mistake, reverted it to `WhenAll` now.